src/components/SummaryCard/SummaryCardAction/SummaryCardAction.js

```diff
@@ -83,6 +83,8 @@ class SummaryCardAction extends Component {
                       iconDescription={closeButtonIconDescription}
                       onClick={this.toggleOpen}
                       kind="ghost"
+                      title={closeButtonIconDescription}
+                      aria-label={closeButtonIconDescription}
```

Comment:
This is in response to an Axe error for the `SummaryCardAction`. While we can definitely add a new prop to allow folks to customize these values further, the easiest thing to do at the moment is use the `closeButtonIconDescription` value.